nemo/collections/asr/models/label_models.py

```diff
@@ -251,7 +251,6 @@ def output_types(self) -> Optional[Dict[str, NeuralType]]:
             "embs": NeuralType(('B', 'D'), AcousticEncodedRepresentation()),
         }
 
-    @typecheck()
```

Comment:
You can just replace your forwarded last two lines with calling forward_for_export() there. Do that in a later pr